src/lib/elements/forms/inputURL.svelte

```diff
@@ -38,6 +38,12 @@
         error = element.validationMessage;
     };
 
+    const handleInput = () => {
+        if (value === '') {
+            value = null;
+        }
```

Comment:
I don't think there is any difference, but is there a reason why we handle it like this instead of doing
```js
$: if(!value){
value = null
}
```

Reply:
@ArmanNik  I tried it, but on doing this, for some reason the update button stops working. It does not detect a change in the input field that way


https://user-images.githubusercontent.com/20852629/212715431-b26d30c4-f092-4596-8d0e-55c6d8a48a53.mov

Reply:
Ahaahahahah got it :muscle: that's a very weird behavior :laughing: